src/app/core/campaign/flight-preview.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map, mergeMap, first } from 'rxjs/operators';
+import { HalDoc } from 'ngx-prx-styleguide';
+import { AuguryService } from '../augury.service';
+
+@Injectable()
+export class FlightPreviewService {
+  constructor(private augury: AuguryService) {}
+
+  createFlightPreview(
+    params: {
+      name: string;
+      set_inventory_uri: string;
+      startAt: string;
+      endAt: string;
+      totalGoal: number;
+      dailyMinimum: number;
+      zones: string[];
+    },
+    flightDoc?: HalDoc,
+    campaignDoc?: HalDoc
+  ): Observable<HalDoc[]> {
+    if (flightDoc) {
```

Comment:
POST is giving a 404 on these first two, but it's working on the last one on the api root